src/coreclr/clrdefinitions.cmake

```diff
@@ -240,6 +240,8 @@ function(set_target_definitions_to_custom_os_and_arch)
     elseif (TARGETDETAILS_ARCH STREQUAL "x86")
       target_compile_definitions(${TARGETDETAILS_TARGET} PRIVATE UNIX_X86_ABI)
     elseif (TARGETDETAILS_ARCH STREQUAL "arm64")
+    elseif (TARGETDETAILS_ARCH STREQUAL "loongarch64")
+      target_compile_definitions(${TARGETDETAILS_TARGET} PRIVATE UNIX_LOONGARCH64_ABI)
```

Comment:
I do not think that `UNIX_LOONGARCH64_ABI` is useful. There is one and only LoongArch64 ABI so far, so we do not need this define. `TARGET_LOONGARCH64` will work just fine instead.

The `*_ABI` defines are only added as needed for OS-specific ABI differences. For example, there is `OSX_ARM64_ABI` since the OSX ARM64 ABI is different from ARM64 ABI everywhere else (WIndows, Linux, ...). However, there is no `UNIX_ARM64_ABI` ABI since it is not needed.

Reply:
OK, I will revert it.